docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics.md

```diff
@@ -33,8 +33,7 @@ When you configure a CloudWatch source, you have the option of defining *AWS tag
 Tag filtering is only supported for user-defined AWS tags, not for AWS-generated tags.
 :::
 
-Here’s how tag filtering works:
-
+Here’s how tag filtering works.
 * If you don't specify a tag filter for a namespace, the source will collect all metrics for the namespace.
```

Comment:
```suggestion
*Specify a tag filter for a namespace **OR** else the source will collect **all** metrics for the namespace.
```

Reply:
Given the severity of the impact, make what they should do a command, and tell them why.